### PR TITLE
~ Added examples of node labels and xref link, per BZ1647168

### PR DESCRIPTION
**Pull request by jhoyt-rh**
@zhouying7780, If you are not the correct person to provide technical review, please let me know

/openshift-docs/install_config/aggregate_logging.adoc

### CODE REVIEW

**Comment by vikram-redhat** [comment]
Hi @jhoyt-rh - this should not have been merged before getting QE and peer review.

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -857,6 +857,17 @@ spec:
...
+For example, if your deployment has three infrastructures, you could add labels for those
+infrastructures as follows:
+----
+oc label node <nodename1> logging-es-node=1
+oc label node <nodename2> logging-es-node=2
+oc label node <nodename3> logging-es-node=3
+----
+
+For information about adding a label to an infrastructure, see
+xref:../admin_guide/manage_nodes.html#updating-labels-on-nodes[Updating Labels on Nodes].
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply install_config/aggregate_logging.adoc]: @vikram-redhat @jhoyt-rh 
Sorry. My fault. I gave Joan incorrect information.

**Pull request merged**